src/scss/apps.scss

```diff
@@ -241,6 +241,7 @@
     width: 32px;
     text-align: center;
     line-height: 20px;
+    margin: 4px;
```

Comment:
Adding in this margin it now looks like this.

![image](https://github.com/user-attachments/assets/da442bc7-dc54-4836-8c94-cd8097342179)